cmd/otel-allocator/allocation/allocator.go

```diff
@@ -2,13 +2,30 @@ package allocation
 
 import (
 	"fmt"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
 	"net/url"
 	"sync"
 
 	"github.com/go-logr/logr"
 	"github.com/prometheus/common/model"
 )
 
+var (
+	collectorsAllocatable = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "allocator_collectors_allocatable",
```

Comment:
I would prefer `opentelemetry_allocator_*` instead or are there other components which use this namespace scheme?

Reply:
i think this is fine, thank you!